BLL/Drink.cs

```diff
@@ -1,11 +1,18 @@
 ﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace BLL
 {
     public class Drink
     {
+        [Key]
+        [DatabaseGenerated(DatabaseGeneratedOption.None)]
         public string Id { get; set; }
+
+        [Required]
```

Comment:
I would think about more restrictions over this entity. I think it will not only be the Name that is required, and you can think about more general restrictions, like min-max length, etc.